CMakeLists.txt

```diff
@@ -146,26 +146,70 @@ install(FILES ${CMAKE_CURRENT_BINARY_DIR}/src/OpenShotVersion.h
 
 ################### DOCUMENTATION ###################
 # Find Doxygen (used for documentation)
-set(DOCS_ENABLED FALSE) # Only set true if Doxygen is found and configured
-if (ENABLE_DOCS)
-  include(cmake/Modules/UseDoxygen.cmake)
+set(LIB_DOCS_ENABLED FALSE) # Only set true if Doxygen is found and configured
```

Comment:
@ferdnyc I'm a bit confused by this logic. We define ENABLE_DOCS, DOCS_ENABLED, LIB_DOCS_ENABLED, AND ENABLE_LIB_DOCS. Just wanted to verify this was the intention.

Reply:
No, I hear you, it is a mess.

`DOCS_ENABLED` and `ENABLE_DOCS` are no more, they've been replaced with `ENABLE_LIB_DOCS` and `LIB_DOCS_ENABLED`. So, there are still only two, they're just more specific.

Previously, we were using the same names in the `libopenshot-audio` build — which interferes with being able to build them together (with `-audio` as an `add_subdirectory()` of the libopenshot build), since the option selected affects both. (To say nothing of the mess that would happen with `DOCS_ENABLED`, when it was being set from both build trees.)

Now, libopenshot uses `ENABLE_LIB_DOCS` and libopenshot-audio uses `ENABLE_AUDIO_DOCS`, so the user has two distinct commandline options that don't interfere with each other. And the internal variables are similarly non-clashing.

The fact that there are even just _two_ similar-seeming names admittedly isn't great, still, though there is "logic" to it: 
* `ENABLE_DOCS` (now `ENABLE_LIB_DOCS`) is the user option that can be set false to disable the documentation build.
* `DOCS_ENABLED` (now `LIB_DOCS_ENABLED`) is the internal status variable that records whether the documentation part of the build should actually be run. If Doxygen or Python3 isn't found by the build system, `LIB_DOCS_ENABLED` will be set to `FALSE` even if the user specified `ENABLE_LIB_DOCS=1` initially, and the `doc` target won't be created.

I suppose I could change the name of `LIB_DOCS_ENABLED` to something like `LIB_DOCS_AVAILABLE`, `LIB_DOCS_ACTIVE`, `LIB_CAN_BUILD_DOCS`, etc... that would hopefully make things a bit less confusing.

Reply:
(Same reason I changed a bunch of strings to use `${PROJECT_NAME}` instead of hardcoding some fixed text. That way, when a combined summary is displayed by a superproject build, it will clearly differentiate items that pertain to `libopenshot` from the ones that pertain to `libopenshot-audio`.)